README.md

```diff
@@ -60,14 +74,15 @@ prowler <provider>
 ```
 ![Prowler CLI Execution](docs/img/short-display.png)
 
+
 ## Prowler Dashboard
 
 ```console
 prowler dashboard
 ```
 ![Prowler Dashboard](docs/img/dashboard.png)
 
-It contains hundreds of controls covering CIS, NIST 800, NIST CSF, CISA, RBI, FedRAMP, PCI-DSS, GDPR, HIPAA, FFIEC, SOC2, GXP, AWS Well-Architected Framework Security Pillar, AWS Foundational Technical Review (FTR), ENS (Spanish National Security Scheme) and your custom security frameworks.
+# Prowler at a Glance
```

Comment:
For this section please use https://hub.prowler.com, we can now remove tables and numbers since it is our source of truth.